src/styles/styles.js

```diff
@@ -1207,9 +1207,9 @@ const styles = {
         paddingVertical: 16,
     },
 
-    chatContentEmpty: {
-        paddingTop: 16,
-        paddingBottom: 16,
+    chatContentCreated: {
```

Comment:
I don't think I understand why this is named "chatContentCreated" - can you elaborate?

Reply:
We are moving from a room header that is shown in the case of an empty state to a header which represents the created state and hence is persistently there.I thought it can be more relevant to call it this way now that we have changed the way of representation. I can revert it, if it is not suitable for the scenario.

Reply:
Maybe something like `chatContentEmptyState` makes more sense?

Reply:
Sure, that will be good!